gslb/cache/controller_obj_cache.go

```diff
@@ -52,6 +52,7 @@ type AviHmObj struct {
 	UUID             string
 	Type             string
 	CloudConfigCksum uint32
+	Description      string
```

Comment:
Since we have to parse the description anyway from the HM object, why can't we make the Description a struct here instead and store the parsed values?